view/frontend/web/instantsearch.js

```diff
@@ -586,6 +586,33 @@ requirejs(['algoliaBundle', 'Magento_Catalog/js/price-utils'], function (algolia
 			};
 		}
 
+		if (!algoliaConfig.autocomplete.enabled) {
+			var customSearchBox = algoliaBundle.instantsearch.connectors.connectSearchBox(function(renderOptions, isFirstRendering) {
+				if (isFirstRendering) {
+					var input = document.querySelector(instant_selector);
+					var clearBtn = input.parentElement.getElementsByClassName('clear-query-autocomplete')[0];
```

Comment:
Is there a possibility that user could've changed the template around the clear button? (meaning `clearBtn` could be `undefined`?)

Reply:
Could be possible, yes. I'll add a condition to check around it!